Vic2ToHoI4/Data_Files/blankMod/output/common/ideas/_cultures.txt

```diff
@@ -3,97 +3,145 @@ ideas = {
 		culture_aborigine = {
 			removal_cost = -1
 			allowed = { always = no } allowed_civil_war = { always = yes }
-			picture = culture_aborigine
+			picture = culture_aborigine	
+			modifier = { }
+		}
+		culture_adelphic = {
+			removal_cost = -1
+			allowed = { always = no } allowed_civil_war = { always = yes }
+			picture = culture_adelphic	
 			modifier = { }
 		}
 		culture_afar = {
 			removal_cost = -1
 			allowed = { always = no } allowed_civil_war = { always = yes }
-			picture = culture_afar
+			picture = culture_afar	
+			modifier = { }
+		}
+		culture_afer = {
+			removal_cost = -1
+			allowed = { always = no } allowed_civil_war = { always = yes }
+			picture = culture_afer	
 			modifier = { }
 		}
-		culture_africaine = {
+		culture_africaine  = {
 			removal_cost = -1
 			allowed = { always = no } allowed_civil_war = { always = yes }
-			picture = culture_africaine
+			picture = culture_africaine 	
```

Comment:
There's a bunch of new tabs and spaces at the end of lines. Please use Notepad++'s options to remove them.

Reply:
Solved